src/Cities/App/Request/StoreCityFormRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Cities\App\Request;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Lightit\Cities\Domain\DataTransferObjects\CityDto;
+
+class StoreCityFormRequest extends FormRequest
+{
+    public const NAME = 'name';
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function rules(): array
+    {
+        return [
+            self::NAME => ['required', 'unique:cities,name'],
+        ];
```

Comment:
You can just do:

'name' => ['required', 'unique:cities,name'],

And not declare the `NAME`

Reply:
This confused me while reading our Slite documentation for the FormRequest classes.

On [Usar los metodos tipados de la clase Request en lugar de solo input](https://lightit.slite.com/app/docs/flX6ZukuAlkM95) constants are declared to mention request parameters. But on 
[Siempre usar Form Request](https://lightit.slite.com/app/docs/eqTcnCyInmSBoW) constants are not used.

In my opinion they are not really helpful since the scope of the constants is only for the  Form Request file, and are not heavily referenced (see `StoreCityFormRequest.php` and `UpdateCityFormRequest.php`), so they could be used directly

@sojeda what do you think?

Reply:
Nice one! My bad totally, I didn't know!